string nodeId
string purposeId
string nodeTopic
string description
